src/web/components/QuoteIcon.tsx

```diff
@@ -1,84 +1,21 @@
 import { css } from '@emotion/react';
 
-import { until } from '@guardian/src-foundations/mq';
+import { QuoteIcon as SourceQuoteIcon } from '@guardian/src-ed-quote-icon';
 
-const quoteStyles = (colour?: string) => css`
-	height: 17px;
-	width: 9px;
-	margin-right: 12px;
-	transform: translateY(-1px);
-	overflow: visible;
-	fill: ${colour && colour};
-`;
-
-const sizeStyles = (size: SmallHeadlineSize) => {
-	const tinySvg = css`
-		svg {
-			height: 13px;
-			width: 7px;
-		}
-	`;
-	const smallSvg = css`
-		svg {
-			height: 16px;
-			width: 8px;
-		}
-	`;
-	const mediumSvg = css`
-		margin-right: 4px;
-		svg {
-			height: 20px;
-			width: 11px;
-		}
-	`;
-	const largeSvg = css`
-		margin-right: 8px;
-		svg {
-			height: 24px;
-			width: 13px;
-		}
-	`;
-	switch (size) {
-		case 'tiny':
-			return css`
-				${tinySvg}
-			`;
-		case 'small':
-			return css`
-				${smallSvg}
-			`;
-		case 'medium':
-			return css`
-				${mediumSvg}
-				${until.desktop} {
-					${smallSvg}
-				}
-			`;
-		case 'large':
-			return css`
-				${largeSvg}
-				${until.desktop} {
-					${mediumSvg}
-				}
-			`;
+const quoteBaseStyles = (colour?: string) => css`
+	span > svg {
+		fill: ${colour && colour};
```

Comment:
this is difficult, in `@guardian/src-ed-quote-icon` we have logic to set QuoteIcon; however, we have a much more complex set of logic for how we choose colours for quote icons (look at `decidePalette.ts`)

Should we extend `QuoteIcon` to allow us to plug in colour? Or should we try to port over our logic to Source? 
Not sure what is best, but this is a current work around to the issue

Reply:
Would this not fall back to `fill: false` in the css if there was no colour?